tests/llmcompressor/transformers/compression/run_compressed_configs/w8a8.yaml

```diff
@@ -1,3 +1,3 @@
 cadence: "commit"
 test_type: "regression"
-model_stub: "nm-testing/tinyllama-w4a16-compressed"
\ No newline at end of file
+model_stub: "nm-testing/tinyllama-w8a8-compressed"
```

Comment:
was this change related to this PR or just something you found?

Reply:
Just another typo I found